### PR TITLE
Bug 2055019: Add imagePullPolicy verification

### PR DESCRIPTION
**Pull request by shaior**
Automates https://bugzilla.redhat.com/show_bug.cgi?id=2055019

### CODE REVIEW

File: functests/1_performance/performance.go [hunk 1]
```diff
@@ -1079,6 +1079,18 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			})
 		})
 	})
+
+	Context("Verify containers specifications", func() {
+
+		// Automates https://bugzilla.redhat.com/show_bug.cgi?id=2055019
```
Comment (marioferh) [inline functests/1_performance/performance.go:1085]: I think this is not neccesary

**Review by marioferh — COMMENTED** [review]
the branch is master, in which version we need to fix it?

File: functests/1_performance/performance.go (hunk 1 above)
Comment (shaior) [reply functests/1_performance/performance.go:1085]: Removed in the new PR target 4.10 branch.
https://github.com/openshift-kni/performance-addon-operators/pull/934

Thank you.

**Comment by shaior** [comment]
New PR targeting 4.10 branch 
https://github.com/openshift-kni/performance-addon-operators/pull/934

**Pull request closed without merging**